lib/index.js

```diff
@@ -44,10 +44,7 @@ rpc.on('session add', data => {
   store_.dispatch(sessionActions.addSession(data));
 });
 
-rpc.on('session data', d => {
-  // the uid is a uuid v4 so it's 36 chars long
-  const uid = d.slice(0, 36);
-  const data = d.slice(36);
```

Comment:
@chabou Why was this done instead of using an object?  Let me know if you want me to revert this part of the PR (I have no idea why to encode the uid + data in this way, but there may be a reason).